Jamulus.pro

```diff
@@ -1,11 +1,21 @@
-VERSION = 3.6.2git
+VERSION = 3.6.2
```

Comment:
Didn't really follow the discussion. Is this intended?

Reply:
See #883  for discussing this

Reply:
Can we resolve this? Does the changelog detection work?

Reply:
I think replacing "git" with "dev" was an accepted solution

Reply:
Ok. Does this break the changelog detection? I don't think so.